var/spack/repos/builtin/packages/openimageio/package.py

```diff
@@ -8,14 +8,15 @@
 
 
 class Openimageio(CMakePackage):
-    """OpenImageIO is a library for reading and writing images, and a bunch of
-    related classes, utilities, and applications."""
+    """Reading, writing, and processing images in a wide variety of file formats, using
+    a format-agnostic API, aimed at VFX applications."""
 
     homepage = "https://www.openimageio.org"
-    url = "https://github.com/OpenImageIO/oiio/archive/Release-1.8.15.tar.gz"
+    url = "https://github.com/AcademySoftwareFoundation/OpenImageIO/archive/refs/tags/v2.5.14.0.tar.gz"
 
     license("Apache-2.0")
 
+    version("2.5.14.0", sha256="0e74372c658f083820872311d126867f10d59b526a856672746de7b2c772034d")
     version("2.2.7.0", sha256="857ac83798d6d2bda5d4d11a90618ff19486da2e5a4c4ff022c5976b5746fe8c")
     version("1.8.15", sha256="4d5b4ed3f2daaed69989f53c0f9364dd87c82dc0a09807b5b6e9008e2426e86f")
```

Comment:
Change of url results in previous versions having invalid checksums.

Reply:
I have updated all checksums.